Assets/MRTK/Examples/Demos/ReadingMode/Scripts/ReadingModeSceneBehavior.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.MixedReality.Toolkit;
+using Microsoft.MixedReality.Toolkit.UI;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.XR;
+
+/// <summary>
+/// Logic for the ReadingModeScene
+/// </summary>
+public class ReadingModeSceneBehavior : MonoBehaviour
```

Comment:
This class is missing a namespace, which is triggering the PR validation failure:

```
WARNING: F:\AIPMRHosted5\_work\18\s\Assets\MRTK\Examples\Demos\ReadingMode\Scripts\ReadingModeSceneBehavior.cs is 
missing a namespace declaration (i.e. missing namespace Microsoft.MixedReality.Toolkit.*)
```